functions/package.json

```diff
@@ -37,7 +37,8 @@
     "object.entries": "^1.0.4",
     "raven": "^2.6.0",
     "replaceall": "^0.1.6",
-    "supports-color": "^5.4.0"
+    "supports-color": "^5.4.0",
```

Comment:
hm, how do you use this lib here?

Reply:
Hmm I'm not sure how that ended up here! I believe that's an atom module I use for formatting in the editor. That can come out.